firefox_puppeteer/ui/menu.py

```diff
@@ -45,6 +61,15 @@ def select(self, label, item):
         """
         return self.get_menu(label).select(item)
 
+    def select_by_id(self, menu_id, item_id):
+        """Select an item in a menu.
+
+        :param menu_id: The ID of the menu, e.g., **file-menu** or **view-menu**.
+        :param item_id: The ID of the item in the menu, e.g.,
+        **menu_newNavigatorTab**.
+        """
+        return self.get_menu_by_id(menu_id).select_by_id(item_id)
+
     class MenuElement(DOMElement):
         """Wraps a menu element DOM element."""
```

Comment:
Best would be to have that as in independent class outside of the menubar class similar to other UIBaseLib based classes. For example see the deck class and its other classes in https://github.com/mozilla/firefox-ui-tests/blob/mozilla-central/firefox_puppeteer/ui/about_window/deck.py.